docs/standard/microservices-architecture/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -87,40 +87,37 @@ services.AddAuthorization(options =>
+In the previous example, the first AddPolicy call is just an alternative way of authorizing by role. If `[Authorize(Policy="AdministratorsOnly")]` is applied to an API, only users in the Administrator role will be able to access it.
 
-The second AddPolicy call demonstrates an easy way to require that a particular claim should be present for the user. The RequireClaim method also optionally takes expected values for the claim. If values are specified, the requirement is met only if the user has both a claim of the correct type and one of the specified values. If you are using the JWT bearer authentication middleware, all JWT properties will be available as user claims.
+The second `AddPolicy` call demonstrates an easy way to require that a particular claim should be present for the user. The `RequireClaim` method also optionally takes expected values for the claim. If values are specified, the requirement is met only if the user has both a claim of the correct type and one of the specified values. If you are using the JWT bearer authentication middleware, all JWT properties will be available as user claims.
 
-The most interesting policy shown here is in the third AddPolicy method, because it uses a custom authorization requirement. By using custom authorization requirements, you can have a great deal of control over how authorization is performed. For this to work, you must implement these types:
+The most interesting policy shown here is in the third `AddPolicy` method, because it uses a custom authorization requirement. By using custom authorization requirements, you can have a great deal of control over how authorization is performed. For this to work, you must implement these types:
 
--   A Requirements type that derives from IAuthorizationRequirement and that contains fields specifying the details of the requirement. In the example, this is an age field for the sample MinimumAgeRequirement type.
+- A Requirements type that derives from `IAuthorizationRequirement` and that contains fields specifying the details of the requirement. In the example, this is an age field for the sample `MinimumAgeRequirement` type.
 
--   A handler that implements AuthorizationHandler&lt;T&gt;, where T is the type of IAuthorizationRequirement that the handler can satisfy. The handler must implement the HandleRequirementAsync method, which checks whether a specified context that contains information about the user satisfies the requirement.
+- A handler that implements `AuthorizationHandler<T>`, where T is the type of `IAuthorizationRequirement` that the handler can satisfy. The handler must implement the `HandleRequirementAsync` method, which checks whether a specified context that contains information about the user satisfies the requirement.
```

Comment:
Again, xrefs for .NET APIs: `AuthorizationHandler<T>` `IAuthorizationRequirement` and `HandleRequirementAsync`. The `T` in "where T is the type..." should be code-fenced.

Reply:
Added the xrefs, but I think the "where T" is part of the explanation, not the syntax.

Fixed it but I'm not sure about xrefing generics, are the updated references ok?

Reply:
I found out I had to use the fully qualified class name as it shows in the url in the API browser 😊